distributed/scheduler.py

```diff
@@ -2688,10 +2704,14 @@ def transition_queued_processing(self, key: str, stimulus_id: str) -> RecsMsgs:
             assert not ts.actor, f"Actors can't be queued: {ts}"
             assert ts in self.queued
 
-        if ws := self.decide_worker_rootish_queuing_enabled():
-            self.queued.discard(ts)
-            worker_msgs = self._add_to_processing(ts, ws)
-        # If no worker, task just stays `queued`
+        if self.is_rootish(ts):
+            if not (ws := self.decide_worker_non_rootish(ts)):
```

Comment:
```suggestion
            if not (ws := self.decide_worker_rootish_queuing_enabled()):
```
was it intentional to use the same function in both places?

Reply:
typo. you reviewed too soon :)

Reply:
Oh, I never pushed the commit. Sorry